src/framework/FormatChecker.ts

```diff
@@ -47,17 +47,20 @@ export class FormatChecker extends Base {
     const logger = this.bot.logger.child({
       id: randomUUID(),
       type: 'FormatChecker',
-      name: this.name,
+      checkerName: this.name,
     });
 
     if (this.checker instanceof RegExp) {
       if (this.checker.test(message.cleanContent)) return;
     } else if (this.checker(message, logger)) return;
 
-    logger.debug('bad format detected');
-
     const { cleanContent, author } = message;
 
+    logger.info(
+      { username: author.username, tag: author.tag, cleanContent },
```

Comment:
```suggestion
      { tag: author.tag, id: author.id, cleanContent },
```
The username is already visible in `tag`, so there is no need to repeat it imo. (and the ID is important)

Reply:
Is the username always in the tag? It is in my case but I thought you could have a different name

Reply:
the tag is only a \`${author.username}#${author.discriminator}\`.

Reply:
ok, I thought it was the nickname